test/ShallowWrapper-spec.jsx

```diff
@@ -86,6 +90,22 @@ describe('shallow', () => {
         expect(wrapper.context().name).to.equal(context.name);
         expect(wrapper.context('name')).to.equal(context.name);
       });
+
+      itIf(REACT16, 'is not introspectable through context API', () => {
+        const SimpleComponent = (props, context) => (
+          <div>{context.name}</div>
+        );
+        SimpleComponent.contextTypes = { name: PropTypes.string };
+
+        const wrapper = shallow(<SimpleComponent />, { context });
+
+        expect(() => wrapper.context()).to.throw(Error,
```

Comment:
if the entire invocation can't fit on one line, all args need to be on a line by themselves - ie, `Error` needs to be on a separate line. same on line 105.